apps/web/src/pages/studies/index.tsx

```diff
@@ -56,8 +54,19 @@ export default function Studies({
 }: StudiesProps) {
   const router = useRouter()
 
+  const [dashboardLink, setDashboardLink] = useState('')
```

Comment:
Is PR an interim stage in development?
The dashboard link seems to be a constant (from the environment), why does it have side effects?
Is this eventually going to be a dynamic value?